src/file-manager/fm-list-view.c

```diff
@@ -1168,17 +1170,26 @@ key_press_callback (GtkWidget *widget, GdkEventKey *event, gpointer callback_dat
         break;
     case GDK_KEY_Return:
     case GDK_KEY_KP_Enter:
-        if ((event->state & GDK_SHIFT_MASK) != 0)
-        {
-            activate_selected_items_alternate (FM_LIST_VIEW (view), NULL, TRUE);
-        }
-        else
-        {
-            activate_selected_items (FM_LIST_VIEW (view));
-        }
-        handled = TRUE;
+	if (GTK_IS_CELL_EDITABLE (listview->details->editable_widget) &&
+	    (event->state & GDK_SHIFT_MASK) || (event->state & GDK_CONTROL_MASK))
```

Comment:
Let's add one more pair of parentheses around this, as `||` has lower priority than `&&`.
Otherwise using Ctrl-Shift-Enter when **not** editing is completely ignored, while in git master it behaves like Shift-Enter.

I know it sounds minor, but let's keep the existing behavior, especially when the change is meant to go into 1.18 branch as well.